src/python/pants/engine/native.py

```diff
@@ -669,6 +662,26 @@ def decompress_tarball(self, tarfile_path, dest_dir):
     result = self.lib.decompress_tarball(tarfile_path, dest_dir)
     return self.context.raise_or_return(result)
 
+  def init_rust_logging(self, level):
+    print("BL: Pythion process {} initialized logging with level {}".format(os.getpid(), level))
+    return self.lib.init_logging(level)
+
+  def maybe_encode_utf8(self, s):
+    return s.encode("utf-8") if PY3 else bytes(s, 'utf-8')
```

Comment:
Why the difference here? `s.encode("utf-8")` should work the same regardless of Python interpreter, that it encodes to bytes. In Python 3, it will error if it is already bytes, whereas Python 2 will encode no matter what.

In fact, I think we can remove this function to inline `s.encode('utf-8')` everywhere we need it.

Reply:
Done